pymess/backend/__init__.py

```diff
@@ -84,6 +121,15 @@ def create_message(self, recipient, content, related_objects, tag, template, **k
     def is_turned_on_batch_sending(self):
         return False
 
+    def publish_or_retry_message(self, message):
```

Comment:
method is used in batch sending. Is checked if message should be resend. If not state is set to ERROR (final state). It is faster than complicated SQL select because there is used index on state field